src/main/groovy/edu/oregonstate/mist/locations/frontend/resources/LocationResource.groovy

```diff
@@ -109,6 +109,12 @@ class LocationResource extends Resource {
                 return notFound().build()
             }
 
+            // validate ranges of lat and lon
+            if(lat && lon && !(lat >= -90 && lat <= 90 && lon >= -180 && lon <= 180)) {
+                return badRequest("Longitude/latitude out of bounds. Bounds are: " +
+                        "-90 <= latitude <= 90 and -180 <= longitude <= 180").build()
```

Comment:
Returning a bad request here is good, but I think we should keep the error message simple. Something like `Invalid coordinate pair` or similar. While indicating the correct bounds for a coordinate pair is helpful, it's not logic specific to our API (they should reference what valid coordinates are elsewhere).